flutter/packages/duck_router/lib/src/provider.dart

```diff
@@ -79,6 +79,10 @@ class DuckInformationProvider extends RouteInformationProvider
         currentState.location,
       );
 
+      if (stackToGoTo == null || stackToGoTo.isEmpty) {
+        return;
+      }
```

Comment:
todo: I would appreciate a comment here mentioning why we return early here

Reply:
Done